extension/json/json_functions/json_array_append.cpp

```diff
@@ -0,0 +1,40 @@
+#include "json_executors.hpp"
+
+namespace duckdb {
+
+//! Append String or JSON value to an array
+yyjson_mut_val *ArrayAppendJSON(yyjson_mut_val *arr, string_t element, yyjson_alc *alc, Vector &result) {
```

Comment:
This (and all the other operation "implementations") can be static (or maybe even a lambda, although I wouldn't change that now unless you really want to)